docs/faq.md

```diff
@@ -55,6 +55,29 @@ https://docs.docker.com/desktop/
 
 **A:** Click on the **Troubleshooting** tab, then click on **Show Logs**.
 
+<!-- #896 -->
+**Q: How can I enable the dashboard for the Traefik ingress controller?**
+
+**A:** Use the command below with `kubectl` for both macOS and Windows:
+
```

Comment:
The IngressRoute approach didn't work for me. I am not sure why it didn't work though.
The `kubectl port-forward..` command works for me though.. So, we can suggest below command to this FAQ.

Traefik dashboard is not exposed by default, for security concerns. Thus, there are multiple ways to expose the dashboard. For instance, the dashboard access could be achieved through a port-forward.

```
kubectl port-forward -n kube-system $(kubectl -n kube-system get pods --selector "app.kubernetes.io/name=traefik" --output=name) 9000:9000
```
Visit `http://127.0.0.1:9000/dashboard/` in your browser to view the dashboard.

Reply:
Revised as suggested, thank you.